src/core/transaction.js

```diff
@@ -1413,6 +1461,24 @@ Transaction.prototype.summary = function() {
   return txSummary;
 };
 
-exports.Transaction = Transaction;
+Transaction.prototype.addSigner = function(signer) {
+  if (_.isUndefined(this.signers)) {
+    this.signers = [];
+  }
+  this.signers.push({Signer: signer});
```

Comment:
As a convenience, `submit_multisigned` sorts the Signers array before submitting it.  If the transaction is submitted as a blob the Signers must arrive pre-sorted.  This gives the transaction a canonical serialized representation.